examples/getting_started/plot_non_fittable_mixin.py

```diff
@@ -47,6 +47,9 @@ class NonFittableStep(NonFittableMixin, BaseStep):
     Also, make sure that BaseStep is the last step you inherit from.
     Note that we could also define the inverse_transform method in the present object.
     """
+    def __init__(self):
+        BaseStep.__init__(self)
+        NonFittableMixin.__init__(self)
```

Comment:
Good, this is the proper way I want Mixins to be initialized, let's continue like what you've presently done for everything whenever we see a discrepancy with the present technique. 

For instance, the `NonFittableMixin` cannot call super within its `NonFittableMixin.__init__(self)`, and the BaseStep was initialized before so as to counter this problem. As discussed yesterday.